2.adoc

```diff
@@ -72,5 +72,94 @@ Optional description
 . Minus A
 . Minus B
 
+=== LibreOffice Writer
+LibreOffice Writer je multiplatformový textový preprocesor z open-source kancelárského balíka LibreOffice.  
+
+link:+https://www.libreoffice.org/discover/writer/+[https://www.libreoffice.org/discover/writer/]
+
+==== Porovnání rozdílů
+* Nejedná sa webovú komponentu, ale o desktopovú aplikáciu
+* Dokumenty nie je možné písať v LML (Lightweight Markup Language)
+* Zameraný na netechnických užívateľov, nezobrazuje čísla riadkov, nemá syntax highlighting
```

Comment:
~nemá syntax highlighting~ nepracuje se zdrojovym kodem

Reply:
jo, myslel som skôr na vloženie kúsku kódu do dokumentu. NIečo takéto
 
![screen shot 2017-11-28 at 20 24 59](https://user-images.githubusercontent.com/6961901/33339679-3f06f492-d47a-11e7-81eb-8b3da3457aba.png)

Reply:
Ok, doplnil bych to tam (_nema syntax highligting (pokud dokument obsahuje snippety zdrojoveho kodu)_)
Bez toho to vypada jako ze dostava Writer zjeba za to, ze nezvyraznuje markup syntax, kdyz vubec nic takovyho nema :D

Reply:
ale pri vkladani textu, e.g. z webu si dokaze obcas zachovat formatovani